dds/DCPS/JsonValueWriter.h

```diff
@@ -308,26 +316,38 @@ void JsonValueWriter<Buffer>::write_enum(const char* name,
 template<typename T>
 std::string to_json(const T& sample)
 {
-  JsonValueWriter<> jvw;
+  rapidjson::StringBuffer buffer;
+  rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
+  JsonValueWriter<rapidjson::Writer<rapidjson::StringBuffer>> jvw(writer);
```

Comment:
Issue with older compilers and ``>>`` for nested templates... here and line 340.